File: app/src/main/java/com/mmjang/ankihelper/data/quote/QuoteDatabaseHelper.java
package com.mmjang.ankihelper.data.quote;

import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;

public class QuoteDatabaseHelper extends SQLiteOpenHelper {
    private static final String DATABASE_NAME = "quote.db";
    private static final int DATABASE_VERSION = 1;

    public QuoteDatabaseHelper(Context context) {
        super(context, DATABASE_NAME, null, DATABASE_VERSION);
    }

    @Override
    public void onCreate(SQLiteDatabase db) {
        // You don't need to create the table here because the database is already pre-populated.
    }

    @Override
    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
        // Handle database schema updates if needed.
    }
}

File: app/src/main/java/com/lmyby/ankiquicker/data/quote/QuoteDatabaseHelper.kt
package com.lmyby.ankiquicker.data.quote

import android.content.Context
import android.database.sqlite.SQLiteDatabase
import android.database.sqlite.SQLiteOpenHelper

/**
 * Converted to Kotlin as part of Phase 2 data model migration
 */
class QuoteDatabaseHelper(context: Context) : SQLiteOpenHelper(context, DATABASE_NAME, null, DATABASE_VERSION) {

    override fun onCreate(db: SQLiteDatabase) {
        // You don't need to create the table here because the database is already pre-populated.
    }

    override fun onUpgrade(db: SQLiteDatabase, oldVersion: Int, newVersion: Int) {
        // Handle database schema updates if needed.
    }

    companion object {
        private const val DATABASE_NAME = "quote.db"
        private const val DATABASE_VERSION = 1
    }
}
